src/components/HelpMark/HelpMark.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import {CircleQuestion} from '@gravity-ui/icons';
+
+import {Icon} from '../Icon';
+import {Popover} from '../Popover';
+import type {PopupPlacement} from '../Popup';
+import type {QAProps} from '../types';
+import {block} from '../utils/cn';
+
+import './HelpMark.scss';
+
+const b = block('help-mark');
+const ICON_SIZE = 16;
+
+export interface HelpMarkProps extends QAProps {
+    buttonProps?: React.ButtonHTMLAttributes<HTMLButtonElement>;
+    buttonRef?: React.RefObject<HTMLButtonElement>;
+    delayClosing?: number;
+    placement?: PopupPlacement;
+    className?: string;
+    children?: React.ReactNode;
+}
+
+export function HelpMark({
+    buttonRef,
+    buttonProps = {},
+    children,
+    className,
+    delayClosing = 300,
+    ...rest
+}: HelpMarkProps) {
+    return (
+        <Popover
+            {...rest}
+            delayClosing={delayClosing}
+            className={b(null, className)}
+            content={children}
+        >
+            <button
```

Comment:
Can we use Popover's render function children here? That would not add extra element